python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py

```diff
@@ -231,3 +264,46 @@ def no_messages_debug_text(self) -> str:
             " PipesDbfsMessageWriter to be explicitly passed to open_dagster_pipes in the external"
             " process."
         )
+
+
+class PipesDbfsStdioReader(PipesBlobStoreStdioReader):
```

Comment:
Should it be `PipesDbfsTextIOReader`? Why `Stdio`?

Reply:
`Stdio` because it derives from `PipesBlobStoreStdioReader`, the purpose of which is to read stdout/stderr streams. Wouldn't be `TextIO` because it doesn't operate on an arbitrary `TextIO` object (it does _write_ to one, which should typically be `sys.stdout` or `sys.stderr`.